main.py

```diff
@@ -94,6 +94,20 @@ def main(cl_arguments):
     cl_args = handle_arguments(cl_arguments)
     args = config.params_from_file(cl_args.config_file, cl_args.overrides)
 
+    # prevent loading old config arg names
```

Comment:
Can you move this block to a helper function?

Also might be simpler to just loop through `name_dict.items()`, like:
```
for old_name, new_name in name_dict.items():
    assert_for_log(old_name not in args, "Error ...")
```